docs/Contributing/responsibilities/README.md

```diff
@@ -0,0 +1,21 @@
+# Fleet Responsibilities Documentation
+
+This directory contains responsibility documents for Fleet product groups.
+
+## Product Group Responsibilities
```

Comment:
This feels like it should be in the product groups dir, probably at the top-level readme? That's one place where a longer doc likely makes sense, so if e.g. responsibilities move it can just be a diff of a single file.

Reply:
This is important to call out separately because these are critical responsibilities of the product groups. For example, the MDM product group is responsible for keeping all of our certificates up-to-date. If they don't, all Fleet MDM enrollments could start failing. Something with that level of business impact we want to highlight in a special directory. 

So far, Software and Orchestration don't have those type of responsibilities clearly documented, but I'd like to get them in here, as well.